src/plugins/intel_gna/src/pre_post_process/preprocessing.cpp

```diff
@@ -32,17 +32,324 @@ int8_t ConvertFloatToInt8(float src) {
+                   bool isGnaDevice,
+                   bool isAvx2Supported) {
+    switch (input_precision) {
+    case Precision::U8:
+    case Precision::I8: {
+        auto src = reinterpret_cast<const uint8_t*>(ptr_src);
+        if (!input_low_precision) {
+            auto dst = reinterpret_cast<int16_t*>(ptr_dst);
+            copy_input_data(dst,
+                            src,
+                            num_frames,
+                            num_group,
+                            num_vector_elements,
+                            num_vector_stride,
+                            orientation,
+                            scaleFactor,
+                            input_low_precision);
+        } else {
+            auto dst = reinterpret_cast<int8_t*>(ptr_dst);
+            copy_input_data(dst,
+                            src,
+                            num_frames,
+                            num_group,
+                            num_vector_elements,
+                            num_vector_stride,
+                            orientation,
+                            scaleFactor,
+                            input_low_precision);
+        }
+        break;
+    }
+    case Precision::I16: {
+        auto src = reinterpret_cast<const int16_t*>(ptr_src);
+        if (!input_low_precision) {
+            auto dst = reinterpret_cast<int16_t*>(ptr_dst);
+            copy_input_data(dst,
+                            src,
+                            num_frames,
+                            num_group,
+                            num_vector_elements,
+                            num_vector_stride,
+                            orientation,
+                            scaleFactor,
+                            input_low_precision);
+        } else {
+            auto dst = reinterpret_cast<int8_t*>(ptr_dst);
+            copy_input_data(dst,
+                            src,
+                            num_frames,
+                            num_group,
+                            num_vector_elements,
+                            num_vector_stride,
+                            orientation,
+                            scaleFactor,
+                            input_low_precision);
+        }
+        break;
+    }
+    case Precision::FP32: {
+        auto src = reinterpret_cast<const float*>(ptr_src);
+        if (!isGnaDevice) {
+            auto dst = reinterpret_cast<float*>(ptr_dst);
+            copy_input_data(dst,
+                            src,
+                            num_frames,
+                            num_group,
+                            num_vector_elements,
+                            num_vector_stride,
+                            orientation,
+                            scaleFactor,
+                            input_low_precision);
+        } else {
+            bool transpose = orientation == kDnnInterleavedOrientation && num_group > 1 && num_vector_stride > 1;
```

Comment:
I suggest using brackets to make the clause more readable. It allows to reduce cognitive load during working with the code.  
```suggestion
            bool transpose = (orientation == kDnnInterleavedOrientation) && (num_group > 1) && (num_vector_stride > 1);
```